seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumJSEnv.scala

```diff
@@ -0,0 +1,27 @@
+package org.scalajs.jsenv.selenium
+
+import org.scalajs.core.tools.io.VirtualJSFile
+import org.scalajs.core.tools.jsdep.ResolvedJSDependency
+import org.scalajs.jsenv.{AsyncJSEnv, ComJSEnv}
+import org.scalajs.jsenv.{JSRunner, AsyncJSRunner, ComJSRunner}
+
+class SeleniumJSEnv(browser: SeleniumBrowser = Firefox,
+    keepAlive: Boolean = false) extends AsyncJSEnv with ComJSEnv {
```

Comment:
Default parameters are bad for binary resilience.

Consider a private constructor with all options, a public constructor without any option, and `withBrowser()` and `withKeepAlive()` methods.